ros/processor/insights_engine_result_consumer.py

```diff
@@ -200,3 +214,15 @@ def process_report(self, host, platform_metadata, reports, system_metadata, perf
                 ).inc()
                 LOG.error("%s - Unable to add system %s to DB belonging to account: %s and org_id: %s - %s",
                           self.prefix, host['id'], account.account, account.org_id, err)
+
+    def trigger_notification(
+        self, system, account, host, platform_metadata, system_previous_state, system_current_state
+    ):
+        if system_previous_state:
+            if system_current_state not in (SYSTEM_STATES['OPTIMIZED'], system_previous_state[0]):
+                LOG.info(
+                    "%s - New suggestion event triggered for the system: %s belonging" +
+                    "to account: %s (%s) and org_id: %s",
+                    self.prefix, system.inventory_id, account.account, account.id, account.org_id
+                )
```

Comment:
```suggestion
                LOG.info(
                    "%s - Triggering a new suggestion event for the system: %s belonging" +
                    "to account: %s (%s) and org_id: %s",
                    self.prefix, system.inventory_id, account.account, account.id, account.org_id
                )
```
At this point, message is not yet triggered. You can either move this log statement below the `new_suggestion_event` method call or modify this log statement.

I would prefer to log this inside a event_producer where it actually triggers the message.